lisa/sut_orchestrator/qemu/platform.py

```diff
@@ -281,11 +307,29 @@ def _configure_nodes(self, environment: Environment, log: Logger) -> None:
                 vm_disks_dir, f"{node_context.vm_name}-cloud-init.iso"
             )
             node_context.os_disk_base_file_path = qemu_node_runbook.qcow2
+            if self.qemu_platform_runbook.is_host_remote():
+                host = self.qemu_platform_runbook.host
+                new_os_disk_base_file_path = os.path.join(
```

Comment:
For the most part, LISA seems to use the `Path` class instead of strings and `os.path,join`.